Core/include/Acts/Seeding/Neighbour.hpp

```diff
@@ -36,28 +36,24 @@ struct Neighbour {
   /// @param grid The grid containing the space points
   /// @param idx The global index of the bin in the grid
   /// @param lowerBound The lower bound of the allowed space points
-  Neighbour(Acts::SpacePointGrid<external_spacepoint_t>&& grid, std::size_t idx,
-            const float lowerBound) = delete;
+  Neighbour(grid_t&& grid, std::size_t idx, const float lowerBound) = delete;
```

Comment:
Was this meant to prevent aggregate construction? I would've thought this is already inhibited by the presence of a user-defined constructor.

Consider removing this.

Reply:
removed